lib/PaymentIntent.php

```diff
@@ -22,10 +23,10 @@
  * @property string $currency
  * @property string $customer
  * @property string $description
+ * @property mixed $last_payment_error
  * @property bool $livemode
  * @property StripeObject $metadata
  * @property mixed $next_source_action
- * @property string $on_behalf_of
```

Comment:
can you leave that one? It's gated but supported.

Reply:
Done